build/marketplace/MarketPlace.sol

```diff
@@ -22,11 +23,14 @@ contract MarketPlace {
         Notional
     }
 
-    /// markets are defined by a market pair which point to a fixed length array of principal token addresses.
-    /// the principal tokens those addresses represent correspond to their Principals enum value, thus the
-    /// array should be ordered in that way
+    /// markets are defined by a market pair which point to a fixed length array
+    /// of principal token addresses. The principal tokens those addresses
+    /// represent correspond to their Principals enum value, thus the array
+    /// should be ordered in that way
     mapping(address => mapping(uint256 => address[9])) public markets;
 
+    mapping(address => mapping(uint256 => address[9])) public pools;
```

Comment:
oh that looks wierd at first, but yeah, i guess there are 0-8 available "slots" for a pool that is associated with an individual principal in a market

i suppose then the pool can be referenced with the Principals enum the same way a given principal (in a market) can?

Reply:
Yes, that's what I was thinking.